scripts/android/unzip.sh

```diff
@@ -0,0 +1,35 @@
+VERSION=$1
+AAR=$2
```

Comment:
example call in readme.md suggests the other order of args - `bash ./scripts/android/unzip.sh "path/to/the/file.aar"`

Reply:
To be honest reviewing these scripts isn't very important because they things they do will likely be embedded in CI scripts instead.

Reply:
And in general there's no real point reviewing if it doesn't work yet ;)